sunpy/visualization/wcsaxes_compat.py

```diff
@@ -181,6 +181,10 @@ def wcsaxes_heliographic_overlay(axes, grid_spacing=10*u.deg):
     -------
     overlay : wcsaxes overlay
         The overlay object.
+
+    Notes
+    -----
+    Keywords are passed on to the overlay object.
```

Comment:
keywords are passed to `astropy.visualization.wcsaxes.coordinates_map.CoordinatesMap.grid`